packages/react-bindings/src/accessibility/getKeyDownHandlers.ts

```diff
@@ -14,31 +14,32 @@ const rtlKeyMap = {
 /**
  * Assigns onKeyDown handler to the slot element, based on Component's actions
  * and keys mappings defined in Accessibility behavior
- * @param {AccessibilityActionHandlers} componentActionHandlers Actions handlers defined in a component.
- * @param {KeyActions} behaviorKeyActions Mappings of actions and keys defined in Accessibility behavior.
+ * @param {AccessibilityActionHandlers} actionHandlers Actions handlers defined in a component.
+ * @param {KeyActions} behaviorActions Mappings of actions and keys defined in Accessibility behavior.
  * @param {boolean} isRtlEnabled Indicates if Left and Right arrow keys should be swapped in RTL mode.
  */
 const getKeyDownHandlers = (
-  componentActionHandlers: AccessibilityActionHandlers,
-  behaviorKeyActions: KeyActions,
+  actionHandlers: AccessibilityActionHandlers,
+  behaviorActions: KeyActions,
   isRtlEnabled?: boolean,
 ): AccessibilityKeyHandlers => {
+  const componentHandlerNames = Object.keys(actionHandlers)
   const keyHandlers = {}
 
-  if (!componentActionHandlers || !behaviorKeyActions) return keyHandlers
+  if (!actionHandlers || !behaviorActions) return keyHandlers
 
-  for (const componentPart in behaviorKeyActions) {
-    const componentPartKeyAction = behaviorKeyActions[componentPart]
-    const handledActions = _.intersection(
-      _.keys(componentPartKeyAction),
-      _.keys(componentActionHandlers),
+  for (const slotName in behaviorActions) {
+    const behaviorSlotAction = behaviorActions[slotName]
+    const handledActions = Object.keys(behaviorSlotAction).filter(
+      actionName => componentHandlerNames.indexOf(actionName) !== -1,
     )
+
```

Comment:
I refactored this place to avoid Lodash usage in our core package, tests are passing.